lib/hanami/model/migrator/adapter.rb

```diff
@@ -38,15 +40,30 @@ def self.for(connection)
             MySQLAdapter
           else
             self
-          end.new(connection)
+          end.new(connection, configuration)
+        end
+
+        class << self
+          private
+
+          # @since x.x.x
+          # @api private
+          def connection_for(configuration)
+            Sequel.connect(
```

Comment:
You can do this in rom by creating a configuration without creating a container from it